src/Delegate/NotFoundDelegate.php

```diff
@@ -16,6 +16,7 @@
 class NotFoundDelegate implements DelegateInterface
 {
     const TEMPLATE_DEFAULT = 'error::404';
+    const LAYOUT_DEFAULT = 'layout::error';
```

Comment:
The main problem I have with this is that we do not at this time define a `layout::error` template in the skeleton application, only a `layout::default`. As such, I think that should likely be the default in this situation, to allow for backwards compatibility with existing applications.